<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarksWidget</class>
 <widget class="QWidget" name="BookmarksWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin" stdset="0">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="LC::Util::TagsLineEdit" name="FavoritesFilterLine_"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="FavoritesFilterType_">
         <item>
          <property name="text">
           <string>Fixed string</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tags</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="FavoritesFilterCaseSensitivity_">
         <property name="text">
          <string>Case sensitive</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="LC::Poshuku::FavoritesTreeView" name="FavoritesView_">
     <property name="contextMenuPolicy">
      <enum>Qt::ContextMenuPolicy::ActionsContextMenu</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragDropMode::DragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::DropAction::MoveAction</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="OpenInTabs_">
     <property name="text">
      <string>Open in tabs</string>
     </property>
    </widget>
   </item>
  </layout>
  <action name="ActionEditBookmark_">
   <property name="text">
    <string>Edit bookmark...</string>
   </property>
   <property name="toolTip">
    <string>Edit title and tags</string>
   </property>
  </action>
  <action name="ActionDeleteBookmark_">
   <property name="text">
    <string>Delete bookmark</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LC::Util::TagsLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">util/tags/tagslineedit.h</header>
  </customwidget>
  <customwidget>
   <class>LC::Poshuku::FavoritesTreeView</class>
   <extends>QTreeView</extends>
   <header>favoritestreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
